src/core.rs

```diff
@@ -62,6 +81,28 @@ impl Scalar {
     }
 }
 
+impl From<native::cvsys_Scalar> for Scalar {
```

Comment:
I've discussed this matter with several Rust full-time developers.

They suggest either just reexport `cvsys` struct or implement newtype over scalar with getters/setters. Unnessessary copying is considered a bad practice.

I think the former is the best approach.